keps/349-all-or-nothing/README.md

```diff
@@ -279,6 +280,37 @@ const (
 
 ```
 
+### Workload API changes
```

Comment:
This API was not introduced by the All or nothing feature.
This is introduced by https://github.com/kubernetes-sigs/kueue/tree/main/keps/1282-pods-ready-requeue-strategy#workload.